packages/material-ui/src/Box/Box.js

```diff
@@ -1,12 +1,10 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
+import { unstable_extendSxProp as extendSxProp } from '@material-ui/system';
 import styled from '../styles/experimentalStyled';
 
-/**
- * @ignore - do not document.
- */
-const Box = React.forwardRef(function Box(props, ref) {
+const BoxInner = React.forwardRef((props, ref) => {
```

Comment:
I assume we will be able to remove this inner component later down the road.

Reply:
Hopefully yes. I had it inlined first, but I needed to add propTypes on it, that’s why I’ve extracted it..